src/.github/workflows/ci.yml.jinja

```
@@ -1,5 +1,4 @@
-
-name: test
+name: build-test-deploy
```

As this can be a normal string, why not something more human-readable then?
```suggestion
name: Build, Test & Deploy
```

Something like https://github.com/docker-mailserver/docker-mailserver/actions?query=workflow%3A%22Build%2C+Test+%26+Deploy%22 for example